### ISSUE TITLE
Stuck on how to install software when creating bioinformatics environment and updating conda package

### ISSUE THREAD

**Issue by dia-dd** [biostars/biostar-handbook-issues#145]
Hi I've installed conda and am trying to create the bioinformatics environment and am getting the following:

> conda create -y --name bioinfo python=3.6
> Collecting package metadata (current_repodata.json): failed
> 
> CondaHTTPError: HTTP 000 CONNECTION FAILED for url <https://conda.anaconda.org/conda-forge/linux-64/current_repodata.json>
> Elapsed: -
> 
> An HTTP error occurred when trying to retrieve this URL.
> HTTP errors are often intermittent, and a simple retry will get you on your way.
> 'https://conda.anaconda.org/conda-forge/linux-64

I've tried this several times with the same result. What am I missing? BTW I get a similar message if I try to update the conda package.

> ~$ conda update -y -n base conda
> Collecting package metadata (current_repodata.json): failed
> 
> CondaHTTPError: HTTP 000 CONNECTION FAILED for url <https://conda.anaconda.org/conda-forge/linux-64/current_repodata.json>
> Elapsed: -
> 
> An HTTP error occurred when trying to retrieve this URL.
> HTTP errors are often intermittent, and a simple retry will get you on your way.
> 'https://conda.anaconda.org/conda-forge/linux-64'

**Comment by ialbert** [comment]
Is it possible that a firewall on your network blocks traffic? I have tried the 
 
    conda update -y -n base conda
 
just now the process (within 1 minute of you opening the ticket) and all works on two unrelated computers both worked as expected.

Move to a different network or look into what could be blocked.

**Comment by dia-dd** [comment]
> Is it possible that a firewall on your network blocks traffic? 
> Move to a different network or look into what could be blocked.

I turned off my firewall and all pc protection and get the same error.

**Comment by ialbert** [comment]
not the computer firewall but the network firewall, are you within an organization?

**Comment by ialbert** [comment]
what happens if you do a

    curl -s https://conda.anaconda.org/conda-forge/linux-64/current_repodata.json | head
    
basically just trying to access the link that seems to be blocked

**Comment by dia-dd** [comment]
> what happens if you do a

I get the following:

> curl -s https://conda.anaconda.org/conda-forge/linux-64/current_repodata.json | head
{
  "info": {
    "subdir": "linux-64"
  },
  "packages": {
    "21cmfast-3.0.3-py36h1af98f8_1.tar.bz2": {
      "build": "py36h1af98f8_1",
      "build_number": 1,
      "depends": [
        "_openmp_mutex >=4.5",

**Comment by dia-dd** [comment]
Missed your previous reply, I am not with an organization, I'm on my home network.

**Comment by ialbert** [comment]
Ok I found this as 
    
    https://stackoverflow.com/questions/50125472/issues-with-installing-python-libraries-on-windows-condahttperror-http-000-co
    
    this seems to imply that you are running conda on windows terminal!
    
    make sure to run conda on the Linux terminal!

**Comment by ialbert** [comment]
This here also:

https://github.com/conda/conda/issues/6007

their solution is:

    conda config --set ssl_verify no

**Comment by dia-dd** [comment]
I have been using the ubuntu terminal, I tried that solution and still not working. I will try downloading from a different network tomorrow.

**Comment by dia-dd** [comment]
I was able to resolve this issue. Someone from the 

> [conda/conda#6007](https://github.com/conda/conda/issues/6007)

Thank you so much for your help with this. I was able to resolve it but I'm not exactly sure what I did to fix it. I tried the solution you mentioned but it didn't work. Someone in the above thread mentioned that they switched their network from 5G to 2G and it fixed their issue. When I did this, nothing happened. I decided to try updating and restarting my computer and suddenly it worked.
